api/src/main/java/org/pmiops/workbench/db/dao/UserService.java

```diff
@@ -227,6 +227,6 @@ private void logAdminAction(Long targetUserId, Long targetWorkspaceId, String ta
   }
 
   public boolean getContactEmailTaken(String contactEmail) {
-    return (userDao.findUserByContactEmail(contactEmail) != null);
+    return (!userDao.findUserByContactEmail(contactEmail).isEmpty());
```

Comment:
`findUserByContactEmail` can throw an exception - would be nice to protect against that. I think `ProfileController.createAccount` will 500 error here.

Reply:
Ignore the above ... the dao call changed from `User` to `List<User>` which should not throw an error.